pom.xml

```diff
@@ -1322,6 +1332,18 @@
         <artifactId>flink-streaming-java_${scala.compat.version}</artifactId>
         <version>${flink.version}</version>
       </dependency>
+
+      <!-- used by some projects like kafka-protobuf-serializer-->
```

Comment:
confluent versions >= 7.1.x have a dependency convergence error with kotlin libs. They depend on `com.squareup.wire:wire-schema-jvm`, which end up depending on kotlin lib versions 1.7.0 and 1.6.10.
Kotlin tries to follow something close to semantic versioning and AFAIK they follow semantic versioning in stdlib, so I decided to use 1.7.22 instead of 1.6.10.

```
[ERROR] Dependency convergence error for org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.7.0 paths to dependency are:
[ERROR] +-org.apache.pinot:pinot-protobuf:jar:1.1.0-SNAPSHOT
[ERROR]   +-io.confluent:kafka-protobuf-serializer:jar:7.4.0:compile
[ERROR]     +-io.confluent:kafka-protobuf-provider:jar:7.4.0:compile
[ERROR]       +-com.squareup.wire:wire-schema-jvm:jar:4.4.3:compile
[ERROR]         +-com.squareup:kotlinpoet:jar:1.12.0:compile
[ERROR]           +-org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.7.0:compile
[ERROR] and
[ERROR] +-org.apache.pinot:pinot-protobuf:jar:1.1.0-SNAPSHOT
[ERROR]   +-io.confluent:kafka-protobuf-serializer:jar:7.4.0:compile
[ERROR]     +-io.confluent:kafka-protobuf-provider:jar:7.4.0:compile
[ERROR]       +-com.squareup.wire:wire-schema-jvm:jar:4.4.3:compile
[ERROR]         +-org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.6.10:compile
[ERROR] and
[ERROR] +-org.apache.pinot:pinot-protobuf:jar:1.1.0-SNAPSHOT
[ERROR]   +-io.confluent:kafka-protobuf-serializer:jar:7.4.0:compile
[ERROR]     +-io.confluent:kafka-protobuf-provider:jar:7.4.0:compile
[ERROR]       +-com.squareup.wire:wire-runtime-jvm:jar:4.4.3:compile
[ERROR]         +-org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.6.10:compile
[ERROR] and
[ERROR] +-org.apache.pinot:pinot-protobuf:jar:1.1.0-SNAPSHOT
[ERROR]   +-io.confluent:kafka-protobuf-serializer:jar:7.4.0:compile
[ERROR]     +-io.confluent:kafka-protobuf-provider:jar:7.4.0:compile
[ERROR]       +-com.squareup.okio:okio-jvm:jar:3.0.0:compile
[ERROR]         +-org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.5.31:compile
```